legacy/firmware/fsm_msg_common.h

```diff
@@ -46,7 +46,8 @@ bool get_features(Features *resp) {
       config_getLanguage(resp->language, sizeof(resp->language));
   resp->has_label = config_getLabel(resp->label, sizeof(resp->label));
   resp->has_initialized = true;
-  resp->initialized = config_isInitialized();
+  // import is not supported anymore so we require mnemonic
```

Comment:
`imported xprv is not supported` sounds less confusing

Reply:
Done.